### ISSUE TITLE
Error: YAML parse error on cert-manager-approver-policy/templates/deployment.yaml: error converting YAML to JSON: yaml: line 48: mapping values are not allowed in this context

### ISSUE THREAD

**Issue by dxg0716** [cert-manager/approver-policy#180]
I install cert-manager-approver-policy by use helm3，but i have some problems：
------------------------------------------------------------------------------
k8s environment：
kubectl version -o json
{
  "clientVersion": {
    "major": "1",
    "minor": "25",
    "gitVersion": "v1.25.4",
    "gitCommit": "872a965c6c6526caa949f0c6ac028ef7aff3fb78",
    "gitTreeState": "clean",
    "buildDate": "2022-11-09T13:36:36Z",
    "goVersion": "go1.19.3",
    "compiler": "gc",
    "platform": "linux/amd64"
  },
  "kustomizeVersion": "v4.5.7",
  "serverVersion": {
    "major": "1",
    "minor": "25",
    "gitVersion": "v1.25.4",
    "gitCommit": "872a965c6c6526caa949f0c6ac028ef7aff3fb78",
    "gitTreeState": "clean",
    "buildDate": "2022-11-09T13:29:58Z",
    "goVersion": "go1.19.3",
    "compiler": "gc",
    "platform": "linux/amd64"
  }
}

helm version
version.BuildInfo{Version:"v3.10.2", GitCommit:"50f003e5ee8704ec937a756c646870227d7c8b58", GitTreeState:"clean", GoVersion:"go1.18.8"}

------------------------------------------------------------------------------
when i run follow command：
helm template -n cert-manager jetstack/cert-manager-approver-policy -f values.yaml --debug
Error: YAML parse error on cert-manager-approver-policy/templates/deployment.yaml: error converting YAML to JSON: yaml: line 48: mapping values are not allowed in this context
helm.go:84: [debug] error converting YAML to JSON: yaml: line 48: mapping values are not allowed in this context
YAML parse error on cert-manager-approver-policy/templates/deployment.yaml
helm.sh/helm/v3/pkg/releaseutil.(*manifestFile).sort
        helm.sh/helm/v3/pkg/releaseutil/manifest_sorter.go:146
helm.sh/helm/v3/pkg/releaseutil.SortManifests
        helm.sh/helm/v3/pkg/releaseutil/manifest_sorter.go:106
helm.sh/helm/v3/pkg/action.(*Configuration).renderResources
        helm.sh/helm/v3/pkg/action/action.go:165
helm.sh/helm/v3/pkg/action.(*Install).RunWithContext
        helm.sh/helm/v3/pkg/action/install.go:259
main.runInstall
        helm.sh/helm/v3/cmd/helm/install.go:278
main.newTemplateCmd.func2
        helm.sh/helm/v3/cmd/helm/template.go:82
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.5.0/command.go:872
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.5.0/command.go:990
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.5.0/command.go:918
main.main
        helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
        runtime/proc.go:250
runtime.goexit
        runtime/asm_amd64.s:1571

------------------------------------------------------------------------------
I see deployment template：
# Source: cert-manager-approver-policy/templates/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: cert-manager-approver-policy
  labels:
    app.kubernetes.io/name: cert-manager-approver-policy
    helm.sh/chart: cert-manager-approver-policy-v0.5.0
    app.kubernetes.io/instance: release-name
    app.kubernetes.io/version: "v0.5.0"
    app.kubernetes.io/managed-by: Helm
spec:
  replicas: 1
  selector:
    matchLabels:
      app: cert-manager-approver-policy
  template:
    metadata:
      labels:
        app: cert-manager-approver-policy
    spec:
      serviceAccountName: cert-manager-approver-policy
      containers:
      - name: cert-manager-approver-policy
        image: "quay.io/jetstack/cert-manager-approver-policy:v0.5.0"
        imagePullPolicy: IfNotPresent
        ports:
        - containerPort: 10250
        - containerPort: 9402
        readinessProbe:
          httpGet:
            port: 6060
            path: "/readyz"
          initialDelaySeconds: 3
          periodSeconds: 7
        command: ["cert-manager-approver-policy"]
        args:
          - --log-level=1

          - --metrics-bind-address=:9402
          - --readiness-probe-bind-address=:6060

          - --webhook-host=0.0.0.0
          - --webhook-port=10250
          - --webhook-service-name=cert-manager-approver-policy
          - --webhook-ca-secret-namespace=cert-manager
          - --webhook-certificate-dir=/tmp

        **resources:            limits:
              cpu: 200m
              memory: 256Mi
            requests:
              cpu: 100m
              memory: 128Mi**
------------------------------------------------------------------------------
my values.yaml
cat values.yaml 
# -- Number of replicas of approver-policy to run.
replicaCount: 1

image:
  # -- Target image repository.
  repository: quay.io/jetstack/cert-manager-approver-policy
  # -- Target image version tag (if empty, Chart AppVersion will be used)
  tag: ""
  # -- Kubernetes imagePullPolicy on Deployment.
  pullPolicy: IfNotPresent

# -- Optional secrets used for pulling the approver-policy container image.
imagePullSecrets: []

app:
  # -- Verbosity of approver-policy logging.
  logLevel: 1 # 1-5

  # -- Extra CLI arguments that will be passed to the approver-policy process.
  extraArgs: []

  # -- List if signer names that approver-policy will be given permission to
  # approve and deny. CertificateRequests referencing these signer names can be
  # processed by approver-policy. See:
  # https://cert-manager.io/docs/concepts/certificaterequest/#approval
  approveSignerNames:
  - "issuers.cert-manager.io/*"
  - "clusterissuers.cert-manager.io/*"

  metrics:
    # -- Port for exposing Prometheus metrics on 0.0.0.0 on path '/metrics'.
    port: 9402
    # -- Service to expose metrics endpoint.
    service:
      # -- Create a Service resource to expose metrics endpoint.
      enabled: true
      # -- Service type to expose metrics.
      type: ClusterIP
      # -- ServiceMonitor resource for this Service.
      servicemonitor:
        enabled: false
        prometheusInstance: default
        interval: 10s
        scrapeTimeout: 5s
        labels: {}

  readinessProbe:
    # -- Container port to expose approver-policy HTTP readiness probe on
    # default network interface.
    port: 6060

  webhook:
    # -- Host that the webhook listens on.
    host: 0.0.0.0
    # -- Port that the webhook listens on.
    port: 10250
    # -- Timeout of webhook HTTP request.
    timeoutSeconds: 5
    # -- Directory to read and store the webhook TLS certificate key pair.
    certificateDir: /tmp
    # -- Type of Kubernetes Service used by the Webhook
    service:
      type: ClusterIP

# -- Optional extra volume mounts. Useful for mounting custom root CAs
volumeMounts: []
#- name: my-volume-mount
#  mountPath: /etc/approver-policy/secrets

# -- Optional extra volumes.
volumes: []
#- name: my-volume
#  secret:
#    secretName: my-secret

resources:
  # -- Kubernetes pod resource limits for approver-policy.
  limits:
    cpu: 200m
    memory: 256Mi
  # -- Kubernetes pod memory resource requests for approver-policy.
  requests:
    cpu: 100m
    memory: 128Mi

------------------------------------------------------------------------------
my deployment.yaml
cat templates/deployment.yaml 
apiVersion: apps/v1
kind: Deployment
metadata:
  name: {{ include "cert-manager-approver-policy.name" . }}
  labels:
{{ include "cert-manager-approver-policy.labels" . | indent 4 }}
spec:
  replicas: {{ .Values.replicaCount }}
  selector:
    matchLabels:
      app: {{ include "cert-manager-approver-policy.name" . }}
  template:
    metadata:
      labels:
        app: {{ include "cert-manager-approver-policy.name" . }}
    spec:
    {{- with .Values.imagePullSecrets }}
      imagePullSecrets:
        {{- toYaml . | nindent 8 }}
    {{- end }}
      serviceAccountName: {{ include "cert-manager-approver-policy.name" . }}
      affinity:
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
            - matchExpressions:
              - key: devops
                operator: In
                values:
                - "true"
      tolerations:
      - effect: NoSchedule
        key: devops
        operator: Equal
        value: "true"        
      containers:
      - name: {{ include "cert-manager-approver-policy.name" . }}
        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
        imagePullPolicy: {{ .Values.image.pullPolicy }}
        ports:
        - containerPort: {{ .Values.app.webhook.port }}
        - containerPort: {{ .Values.app.metrics.port }}
        readinessProbe:
          httpGet:
            port: {{ .Values.app.readinessProbe.port }}
            path: "/readyz"
          initialDelaySeconds: 3
          periodSeconds: 7
        command: ["cert-manager-approver-policy"]
        args:
          - --log-level={{.Values.app.logLevel}}

          {{- range .Values.app.extraArgs }}
          - {{ . }}
          {{- end  }}

          - --metrics-bind-address=:{{.Values.app.metrics.port}}
          - --readiness-probe-bind-address=:{{.Values.app.readinessProbe.port}}

          - --webhook-host={{.Values.app.webhook.host}}
          - --webhook-port={{.Values.app.webhook.port}}
          - --webhook-service-name={{ include "cert-manager-approver-policy.name" . }}
          - --webhook-ca-secret-namespace={{.Release.Namespace}}
          - --webhook-certificate-dir={{.Values.app.webhook.certificateDir}}

        {{- if .Values.volumeMounts }}
        volumeMounts:
{{ toYaml .Values.volumeMounts | indent 10 }}
        {{- end }}

        resources:
          {{- toYaml .Values.resources | indent 12 }}

      {{- if .Values.volumes }}
      volumes:
{{ toYaml .Values.volumes | indent 6 }}
      {{- end }}
------------------------------------------------------------------------------

Who can help me look at this problem，thanks very much

**Comment by dxg0716** [comment]
**cat values.yaml 
# -- Number of replicas of approver-policy to run.
replicaCount: 1

image:
  # -- Target image repository.
  repository: quay.io/jetstack/cert-manager-approver-policy
  # -- Target image version tag (if empty, Chart AppVersion will be used)
  tag: ""
  # -- Kubernetes imagePullPolicy on Deployment.
  pullPolicy: IfNotPresent

# -- Optional secrets used for pulling the approver-policy container image.
imagePullSecrets: []

app:
  # -- Verbosity of approver-policy logging.
  logLevel: 1 # 1-5

  # -- Extra CLI arguments that will be passed to the approver-policy process.
  extraArgs: []

  # -- List if signer names that approver-policy will be given permission to
  # approve and deny. CertificateRequests referencing these signer names can be
  # processed by approver-policy. See:
  # https://cert-manager.io/docs/concepts/certificaterequest/#approval
  approveSignerNames:
  - "issuers.cert-manager.io/*"
  - "clusterissuers.cert-manager.io/*"

  metrics:
    # -- Port for exposing Prometheus metrics on 0.0.0.0 on path '/metrics'.
    port: 9402
    # -- Service to expose metrics endpoint.
    service:
      # -- Create a Service resource to expose metrics endpoint.
      enabled: true
      # -- Service type to expose metrics.
      type: ClusterIP
      # -- ServiceMonitor resource for this Service.
      servicemonitor:
        enabled: false
        prometheusInstance: default
        interval: 10s
        scrapeTimeout: 5s
        labels: {}

  readinessProbe:
    # -- Container port to expose approver-policy HTTP readiness probe on
    # default network interface.
    port: 6060

  webhook:
    # -- Host that the webhook listens on.
    host: 0.0.0.0
    # -- Port that the webhook listens on.
    port: 10250
    # -- Timeout of webhook HTTP request.
    timeoutSeconds: 5
    # -- Directory to read and store the webhook TLS certificate key pair.
    certificateDir: /tmp
    # -- Type of Kubernetes Service used by the Webhook
    service:
      type: ClusterIP

# -- Optional extra volume mounts. Useful for mounting custom root CAs
volumeMounts: []
#- name: my-volume-mount
#  mountPath: /etc/approver-policy/secrets

# -- Optional extra volumes.
volumes: []
#- name: my-volume
#  secret:
#    secretName: my-secret


resources:
  # -- Kubernetes pod resource limits for approver-policy.
  limits:
    cpu: 200m
    memory: 256Mi
  # -- Kubernetes pod memory resource requests for approver-policy.
  requests:
    cpu: 100m
    memory: 128Mi**

**Comment by dxg0716** [comment]
`# -- Number of replicas of approver-policy to run.
replicaCount: 1

image:
  # -- Target image repository.
  repository: quay.io/jetstack/cert-manager-approver-policy
  # -- Target image version tag (if empty, Chart AppVersion will be used)
  tag: ""
  # -- Kubernetes imagePullPolicy on Deployment.
  pullPolicy: IfNotPresent

# -- Optional secrets used for pulling the approver-policy container image.
imagePullSecrets: []

app:
  # -- Verbosity of approver-policy logging.
  logLevel: 1 # 1-5

  # -- Extra CLI arguments that will be passed to the approver-policy process.
  extraArgs: []

  # -- List if signer names that approver-policy will be given permission to
  # approve and deny. CertificateRequests referencing these signer names can be
  # processed by approver-policy. See:
  # https://cert-manager.io/docs/concepts/certificaterequest/#approval
  approveSignerNames:
  - "issuers.cert-manager.io/*"
  - "clusterissuers.cert-manager.io/*"

  metrics:
    # -- Port for exposing Prometheus metrics on 0.0.0.0 on path '/metrics'.
    port: 9402
    # -- Service to expose metrics endpoint.
    service:
      # -- Create a Service resource to expose metrics endpoint.
      enabled: true
      # -- Service type to expose metrics.
      type: ClusterIP
      # -- ServiceMonitor resource for this Service.
      servicemonitor:
        enabled: false
        prometheusInstance: default
        interval: 10s
        scrapeTimeout: 5s
        labels: {}

  readinessProbe:
    # -- Container port to expose approver-policy HTTP readiness probe on
    # default network interface.
    port: 6060

  webhook:
    # -- Host that the webhook listens on.
    host: 0.0.0.0
    # -- Port that the webhook listens on.
    port: 10250
    # -- Timeout of webhook HTTP request.
    timeoutSeconds: 5
    # -- Directory to read and store the webhook TLS certificate key pair.
    certificateDir: /tmp
    # -- Type of Kubernetes Service used by the Webhook
    service:
      type: ClusterIP

# -- Optional extra volume mounts. Useful for mounting custom root CAs
volumeMounts: []
#- name: my-volume-mount
#  mountPath: /etc/approver-policy/secrets

# -- Optional extra volumes.
volumes: []
#- name: my-volume
#  secret:
#    secretName: my-secret


resources:
  # -- Kubernetes pod resource limits for approver-policy.
  limits:
    cpu: 200m
    memory: 256Mi
  # -- Kubernetes pod memory resource requests for approver-policy.
  requests:
    cpu: 100m
    memory: 128Mi`

**Comment by dxg0716** [comment]
```
# -- Number of replicas of approver-policy to run.
replicaCount: 1

image:
  # -- Target image repository.
  repository: quay.io/jetstack/cert-manager-approver-policy
  # -- Target image version tag (if empty, Chart AppVersion will be used)
  tag: ""
  # -- Kubernetes imagePullPolicy on Deployment.
  pullPolicy: IfNotPresent

# -- Optional secrets used for pulling the approver-policy container image.
imagePullSecrets: []

app:
  # -- Verbosity of approver-policy logging.
  logLevel: 1 # 1-5

  # -- Extra CLI arguments that will be passed to the approver-policy process.
  extraArgs: []

  # -- List if signer names that approver-policy will be given permission to
  # approve and deny. CertificateRequests referencing these signer names can be
  # processed by approver-policy. See:
  # https://cert-manager.io/docs/concepts/certificaterequest/#approval
  approveSignerNames:
  - "issuers.cert-manager.io/*"
  - "clusterissuers.cert-manager.io/*"

  metrics:
    # -- Port for exposing Prometheus metrics on 0.0.0.0 on path '/metrics'.
    port: 9402
    # -- Service to expose metrics endpoint.
    service:
      # -- Create a Service resource to expose metrics endpoint.
      enabled: true
      # -- Service type to expose metrics.
      type: ClusterIP
      # -- ServiceMonitor resource for this Service.
      servicemonitor:
        enabled: false
        prometheusInstance: default
        interval: 10s
        scrapeTimeout: 5s
        labels: {}

  readinessProbe:
    # -- Container port to expose approver-policy HTTP readiness probe on
    # default network interface.
    port: 6060

  webhook:
    # -- Host that the webhook listens on.
    host: 0.0.0.0
    # -- Port that the webhook listens on.
    port: 10250
    # -- Timeout of webhook HTTP request.
    timeoutSeconds: 5
    # -- Directory to read and store the webhook TLS certificate key pair.
    certificateDir: /tmp
    # -- Type of Kubernetes Service used by the Webhook
    service:
      type: ClusterIP

# -- Optional extra volume mounts. Useful for mounting custom root CAs
volumeMounts: []
#- name: my-volume-mount
#  mountPath: /etc/approver-policy/secrets

# -- Optional extra volumes.
volumes: []
#- name: my-volume
#  secret:
#    secretName: my-secret


resources:
  # -- Kubernetes pod resource limits for approver-policy.
  limits:
    cpu: 200m
    memory: 256Mi
  # -- Kubernetes pod memory resource requests for approver-policy.
  requests:
    cpu: 100m
    memory: 128Mi
```

**Comment by dxg0716** [comment]
### this is my value.yaml
```
cat values.yaml 
# -- Number of replicas of approver-policy to run.
replicaCount: 1

image:
  # -- Target image repository.
  repository: quay.io/jetstack/cert-manager-approver-policy
  # -- Target image version tag (if empty, Chart AppVersion will be used)
  tag: ""
  # -- Kubernetes imagePullPolicy on Deployment.
  pullPolicy: IfNotPresent

# -- Optional secrets used for pulling the approver-policy container image.
imagePullSecrets: []

app:
  # -- Verbosity of approver-policy logging.
  logLevel: 1 # 1-5

  # -- Extra CLI arguments that will be passed to the approver-policy process.
  extraArgs: []

  # -- List if signer names that approver-policy will be given permission to
  # approve and deny. CertificateRequests referencing these signer names can be
  # processed by approver-policy. See:
  # https://cert-manager.io/docs/concepts/certificaterequest/#approval
  approveSignerNames:
  - "issuers.cert-manager.io/*"
  - "clusterissuers.cert-manager.io/*"

  metrics:
    # -- Port for exposing Prometheus metrics on 0.0.0.0 on path '/metrics'.
    port: 9402
    # -- Service to expose metrics endpoint.
    service:
      # -- Create a Service resource to expose metrics endpoint.
      enabled: true
      # -- Service type to expose metrics.
      type: ClusterIP
      # -- ServiceMonitor resource for this Service.
      servicemonitor:
        enabled: false
        prometheusInstance: default
        interval: 10s
        scrapeTimeout: 5s
        labels: {}

  readinessProbe:
    # -- Container port to expose approver-policy HTTP readiness probe on
    # default network interface.
    port: 6060

  webhook:
    # -- Host that the webhook listens on.
    host: 0.0.0.0
    # -- Port that the webhook listens on.
    port: 10250
    # -- Timeout of webhook HTTP request.
    timeoutSeconds: 5
    # -- Directory to read and store the webhook TLS certificate key pair.
    certificateDir: /tmp
    # -- Type of Kubernetes Service used by the Webhook
    service:
      type: ClusterIP

# -- Optional extra volume mounts. Useful for mounting custom root CAs
volumeMounts: []
#- name: my-volume-mount
#  mountPath: /etc/approver-policy/secrets

# -- Optional extra volumes.
volumes: []
#- name: my-volume
#  secret:
#    secretName: my-secret


resources:
  # -- Kubernetes pod resource limits for approver-policy.
  limits:
    cpu: 200m
    memory: 256Mi
  # -- Kubernetes pod memory resource requests for approver-policy.
  requests:
    cpu: 100m
    memory: 128Mi
```
### this is my deployment.yaml
```
cat templates/deployment.yaml 
apiVersion: apps/v1
kind: Deployment
metadata:
  name: {{ include "cert-manager-approver-policy.name" . }}
  labels:
{{ include "cert-manager-approver-policy.labels" . | indent 4 }}
spec:
  replicas: {{ .Values.replicaCount }}
  selector:
    matchLabels:
      app: {{ include "cert-manager-approver-policy.name" . }}
  template:
    metadata:
      labels:
        app: {{ include "cert-manager-approver-policy.name" . }}
    spec:
    {{- with .Values.imagePullSecrets }}
      imagePullSecrets:
        {{- toYaml . | nindent 8 }}
    {{- end }}
      serviceAccountName: {{ include "cert-manager-approver-policy.name" . }}
      affinity:
        nodeAffinity:
          requiredDuringSchedulingIgnoredDuringExecution:
            nodeSelectorTerms:
            - matchExpressions:
              - key: devops
                operator: In
                values:
                - "true"
      tolerations:
      - effect: NoSchedule
        key: devops
        operator: Equal
        value: "true"        
      containers:
      - name: {{ include "cert-manager-approver-policy.name" . }}
        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
        imagePullPolicy: {{ .Values.image.pullPolicy }}
        ports:
        - containerPort: {{ .Values.app.webhook.port }}
        - containerPort: {{ .Values.app.metrics.port }}
        readinessProbe:
          httpGet:
            port: {{ .Values.app.readinessProbe.port }}
            path: "/readyz"
          initialDelaySeconds: 3
          periodSeconds: 7
        command: ["cert-manager-approver-policy"]
        args:
          - --log-level={{.Values.app.logLevel}}

          {{- range .Values.app.extraArgs }}
          - {{ . }}
          {{- end  }}

          - --metrics-bind-address=:{{.Values.app.metrics.port}}
          - --readiness-probe-bind-address=:{{.Values.app.readinessProbe.port}}

          - --webhook-host={{.Values.app.webhook.host}}
          - --webhook-port={{.Values.app.webhook.port}}
          - --webhook-service-name={{ include "cert-manager-approver-policy.name" . }}
          - --webhook-ca-secret-namespace={{.Release.Namespace}}
          - --webhook-certificate-dir={{.Values.app.webhook.certificateDir}}

        {{- if .Values.volumeMounts }}
        volumeMounts:
{{ toYaml .Values.volumeMounts | indent 10 }}
        {{- end }}

        resources:
          {{- toYaml .Values.resources | indent 12 }}

      {{- if .Values.volumes }}
      volumes:
{{ toYaml .Values.volumes | indent 6 }}
      {{- end }}
```

**Comment by dxg0716** [comment]
### this is error
```
Error: YAML parse error on cert-manager-approver-policy/templates/deployment.yaml: error converting YAML to JSON: yaml: line 48: mapping values are not allowed in this context
helm.go:84: [debug] error converting YAML to JSON: yaml: line 48: mapping values are not allowed in this context
YAML parse error on cert-manager-approver-policy/templates/deployment.yaml
helm.sh/helm/v3/pkg/releaseutil.(*manifestFile).sort
        helm.sh/helm/v3/pkg/releaseutil/manifest_sorter.go:146
helm.sh/helm/v3/pkg/releaseutil.SortManifests
        helm.sh/helm/v3/pkg/releaseutil/manifest_sorter.go:106
helm.sh/helm/v3/pkg/action.(*Configuration).renderResources
        helm.sh/helm/v3/pkg/action/action.go:165
helm.sh/helm/v3/pkg/action.(*Install).RunWithContext
        helm.sh/helm/v3/pkg/action/install.go:259
main.runInstall
        helm.sh/helm/v3/cmd/helm/install.go:278
main.newTemplateCmd.func2
        helm.sh/helm/v3/cmd/helm/template.go:82
github.com/spf13/cobra.(*Command).execute
        github.com/spf13/cobra@v1.5.0/command.go:872
github.com/spf13/cobra.(*Command).ExecuteC
        github.com/spf13/cobra@v1.5.0/command.go:990
github.com/spf13/cobra.(*Command).Execute
        github.com/spf13/cobra@v1.5.0/command.go:918
main.main
        helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
        runtime/proc.go:250
runtime.goexit
        runtime/asm_amd64.s:1571
```
### helm render deployment.yaml
```
# Source: cert-manager-approver-policy/templates/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: cert-manager-approver-policy
  labels:
    app.kubernetes.io/name: cert-manager-approver-policy
    helm.sh/chart: cert-manager-approver-policy-v0.5.0
    app.kubernetes.io/instance: release-name
    app.kubernetes.io/version: "v0.5.0"
    app.kubernetes.io/managed-by: Helm
spec:
  replicas: 1
  selector:
    matchLabels:
      app: cert-manager-approver-policy
  template:
    metadata:
      labels:
        app: cert-manager-approver-policy
    spec:
      serviceAccountName: cert-manager-approver-policy
      containers:
      - name: cert-manager-approver-policy
        image: "quay.io/jetstack/cert-manager-approver-policy:v0.5.0"
        imagePullPolicy: IfNotPresent
        ports:
        - containerPort: 10250
        - containerPort: 9402
        readinessProbe:
          httpGet:
            port: 6060
            path: "/readyz"
          initialDelaySeconds: 3
          periodSeconds: 7
        command: ["cert-manager-approver-policy"]
        args:
          - --log-level=1

          - --metrics-bind-address=:9402
          - --readiness-probe-bind-address=:6060

          - --webhook-host=0.0.0.0
          - --webhook-port=10250
          - --webhook-service-name=cert-manager-approver-policy
          - --webhook-ca-secret-namespace=cert-manager
          - --webhook-certificate-dir=/tmp

        resources:            limits:
              cpu: 200m
              memory: 256Mi
            requests:
              cpu: 100m
              memory: 128Mi
```
i see resource error format，but i don't know how to change，can help me？thanks very much！！！

**Comment by dxg0716** [comment]
@JoshVanL can you help me？thanks

**Comment by rickymulder** [comment]
@dxg0716 
This looks like an issue that was resolved with PR #168 
What version of the chart are you using?  I know 0.5.0 worked for me.

**Comment by bradfordwagner** [comment]
@dxg0716 The fix #168 went out with 0.6.0. However, the chart hasn't been released yet.

**Comment by inteon** [comment]
the chart should have been released already, please reopen if there is still an issue
